src/API/Helpers/RequestBuilder.php

```diff
@@ -39,9 +39,9 @@ class RequestBuilder
     /**
      * HTTP method to use for the request.
      *
-     * @var array
+     * @var string
      */
-    protected $method = [];
+    protected $method;
```

Comment:
what's the need for this change? Also, couldn't this have potential to be a breaking change for any downstream consumers/extenders?

Reply:
Was a typo that should be corrected. $method should never have been defined as an array — it's expected to be a string representation of the HTTP request method. As it's passed on to Guzzle directly, Guzzle would have thrown an error if anything but a string had been passed. This wouldn't be a breaking change for that reason, it never would have worked if used contrary.